packages/go_router/lib/src/information_provider.dart

```diff
@@ -36,13 +36,12 @@ enum NavigatingType {
 }
 
 /// The data class to be stored in [RouteInformation.state] to be used by
-/// [GoRouteInformationPrarser].
+/// [GoRouteInformationParser].
 ///
 /// This state class is used internally in go_router and will not be send to
 /// the engine.
 class RouteInformationState<T> {
   /// Creates an InternalRouteInformationState.
-  @visibleForTesting
```

Comment:
I had to remove that line because I need to build an instance of `RouteInformationState` in `GoRouter.go()`...